tests/test_files/test_schematron.sch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema xmlns:sch="http://purl.oclc.org/dsdl/schematron" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+  <sch:ns prefix="diggs" uri="http://diggsml.org/schemas/2.5.a"/>
+  <sch:ns prefix="gml" uri="http://www.opengis.net/gml/3.2"/>
+
+<sch:pattern>
+  <sch:rule context="/*">
+    <sch:assert test="name()='Diggs'">Name of root is not 'Diggs'.</sch:assert>
+  </sch:rule>
+</sch:pattern>
+
+<sch:pattern>
+  <sch:rule context="diggs:Project">
+    <sch:assert test="@gml:id='Project_1111-2222-33'">Project ID incorrect.</sch:assert>
+  </sch:rule>
+</sch:pattern>
+
+<sch:pattern>
```

Comment:
Is the `<sch:report> ... </sch:report>` section primarily for testing the `error_log` output? I noticed this gives a "False" when validating the testing Schematron using `sch_schema.validate(parsed_xml_tree)`. The `error_log` also shows a error `...0:0:ERROR:SCHEMATRONV:SCHEMATRONV_ASSERT: ...`